src/test/lib/operators/maintenance/create_table_test.cpp

```diff
@@ -66,6 +85,25 @@ TEST_F(CreateTableTest, Execute) {
   EXPECT_EQ(table->column_definitions(), column_definitions);
 }
 
+TEST_F(CreateTableTest, SoftKeyConstraints) {
+  const auto context = Hyrise::get().transaction_manager.new_transaction_context(AutoCommit::No);
+  const auto input_table = const_pointer_cast<Table>(dummy_table_wrapper->table);
+  const auto unique_constraint = TableKeyConstraint{{ColumnID{0}}, KeyConstraintType::UNIQUE};
+  input_table->add_soft_key_constraint(unique_constraint);
+
+  create_table->set_transaction_context(context);
+
```

Comment:
In `CreateTableTest, NameAndDescriptionWithConstraints`, setting up the table happens first before setting up and performing the transaction. It's only a minor concern, but perhaps, we could do it in a similar way here. Further, `context` is only used in line 94 so it can be moved close to the code location where it's actually used, i.e., above line 94.